### PR TITLE
SEO-191916-ASP.NET-Daterangepicker

### PR DESCRIPTION
**Pull request by mercy-orejo**
Hi @Yvone-Atieno  

|Title|Description|
|--|--|
|Task Category| Site Audit
|Content Task Link/Mail Screenshot| NA|
|UX task| NA|
|Hotfix|https://github.com/syncfusion-content/asp.netwebforms-docs/pull/565
|Ticket/Task link| https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/191916|
|Excel/SharePoint link| NA|

|Changes made|Reason for changes|
|--|--|
|Meta Title| had no platform name and control names according to SEO guideline and Syncfusion
|Meta Description| had less character counts not as per indicated at SEo guodline which states its character should not be less than 100 and not exceed 165
|Meta h1| Was missing platform and control name not as per the SEO guideline
|redirection links| validated links to the redirected links
Regards,
Mercy A.

### CODE REVIEW

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/DatePicker/Display-Format.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-description: Configuring display format of DatePicker
+title: Display Format in ASP.NET Web Forms DateRangePicker | Syncfusion
+description: Learn here all about configuring display format of DatePicker in Syncfusion ASP.NET Web Forms DateRangePicker control, it's elements, and more.
 platform: aspnet
 control: DatePicker
 documentation: ug
 ---
-# Formatting
-
+#  Formatting in ASP.NET DateRangePicker
```
[the remark this answers is not in the document]
Comment (mercy-orejo) [reply aspnet/DatePicker/Display-Format.md:9]: Updated the correct title.

File: aspnet/DatePicker/Display-Format.md [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
-description: Configuring display format of DatePicker
+title: Display Format in ASP.NET Web Forms DatePicker| Syncfusion
+description: Learn here all about configuring display format of DatePicker in Syncfusion ASP.NET Web Forms DatePicker control, it's elements, and more.
 platform: aspnet
 control: DatePicker
 documentation: ug
 ---
-# Formatting
-
+#  Formatting in ASP.NET Web Forms DatePicker
```
Comment (Yvone-Atieno) [inline aspnet/DatePicker/Display-Format.md:9]: Remove extra space

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

**Pull request closed without merging**